### PR TITLE
Added httpbin.org example

### PR DESCRIPTION
**Pull request by leogregianin**
More examples with more HTTP verbs. Issue #5  
Inspired by @dunossauro and @camilamaia

### CODE REVIEW

File: httpbin-api/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# httpbin.org REST API
+
+A simple HTTP Request & Response Service.
```
Comment (camilamaia) [inline httpbin-api/README.md:3]:
```suggestion
[httpbin.org](https://httpbin.org) is a simple HTTP Request & Response Service.
```

**Review by camilamaia — COMMENTED** [review]
Hey, thank you very much for the contribution!! 🙇 It is super good to have more examples here!

Can we have the `scanapi-report.html` file also in this PR? This way people can compare if they are generating reports properly.

Thanks!

File: httpbin-api/README.md (hunk 1 above)
Comment (leogregianin) [reply httpbin-api/README.md]: Good improvements. I made changes.

**Review by camilamaia — APPROVED** [review]
🎉

**Comment by camilamaia** [comment]
@leogregianin I am going to send you an invite to join the org! Everyone that has contributed with a PR merged is invited. Here you can check our [Community Continuity Guidelines ](https://github.com/scanapi/contributors/blob/master/Contributing.md) so you can understand better how it works.

And once again, thank you very much!!

**Comment by leogregianin** [comment]
Thanks @camilamaia. I hope to contribute more.

**Pull request merged**